src/main/java/goodgenerator/blocks/tileEntity/YottaFluidTank.java

```diff
@@ -138,16 +138,11 @@ public boolean addFluid(int amount) {
         }
     }
 
-    private int calGlassTier(int meta) {
-        if (meta >= 1 && meta <= 6) return meta; // returns correct meta for Tiers 1-6, 7-12 is colour variations of HV
-        if (meta >= 7 && meta <= 12) return 1; // For all the HV Glass colour variations
-        return meta; // returns the rest
-    }
-
     @Override
     public boolean checkMachine_EM(IGregTechTileEntity aBaseMetaTileEntity, ItemStack aStack) {
         mStorage = BigInteger.ZERO;
         glassMeta = 0;
```

Comment:
Maybe it would be a good idea to rename this variable to glassTier?

Reply:
I mean you could but iirc its called meta everywhere else